.github/workflows/ci.yml

```diff
@@ -1,8 +1,8 @@
 name: Tests
 
 on:
-  push:
   pull_request:
+  push:
     branches:
       - main
```

Comment:
hmm, that would mean that CI doesn't run when manually pushing a branch like `v1.1`, right?

Ideally, we'd not run the push jobs if there's a PR, but do run for every other case, but not sure if that's easy to do. If not, I'd just keep the current behavior and not care about the defaults?

Reply:
@SteffenDE I've updated the GitHub config and did a trial on my own forked repo and it seems to work on pushing to new branch, see https://github.com/myfloss/phoenix_pubsub/actions/runs/19671611456

And we don't have duplicated jobs for this pull request, see https://github.com/phoenixframework/phoenix_pubsub/pull/203/checks